src/Quadrature/quadrature.jl

```diff
@@ -57,13 +60,17 @@ function QuadratureRule{shape}(weights::Vector{T}, points::Vector{Vec{dim, T}})
     QuadratureRule{shape, T}(weights, points)
 end
 
+@inline _default_quadrature_rule(::RefHypercube) = :legendre
```

Comment:
If we want the order-dependent defaults that we currently have, perhaps add `order` to these functions as well?

Reply:
Fixed. I hope.

Reply:
Not sure. My argument is as follows. If we provide some default rule, then we should stick to it to not introduce behavior which could lead to potentially hard to track down bugs due to different properties in the quadrature rule.